### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by butlermuwo**
## Add basic UI
in this pull request I did the following:
- Created a file app.rb that will serve as your console app entry-point.
- In my main.rb I defined the entry point.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Many thanks to Microverse
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (ahmetbozaci) [inline README.md:62]: - [ ] Please fix the broken link.

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    print 'Age: '
+    age = gets.chomp
+
+    print 'Name: '
+    name = gets.chomp
+
+    student = Student.new(age, name)
+    @people.push(student)
+    puts 'Student Created Successfully'
+  end
```
Comment (ahmetbozaci) [inline app.rb]: - [ ] While creating students ask 'has parent permission' like in the video.

image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @butlermuwo  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.



### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏

almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by rukundoeric — APPROVED** [review]
Hi @butlermuwo,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**